service/service.go

```diff
@@ -107,8 +109,18 @@ func New(tarball io.Reader, options ...Option) (*Service, error) {
 	if err != nil {
 		return nil, err
 	}
+
 	s.injectDefinition(def)
 
+	if err := s.validateConfigurationEnv(env); err != nil {
```

Comment:
missing tests for new behaviour

Reply:
solved by https://github.com/mesg-foundation/core/pull/654/commits/6c7c97734386a0664a0234f2a981191c6083cf02